src/zac/core/forms.py

```diff
@@ -287,3 +286,34 @@ def get_process_variables(self) -> Dict[str, List[str]]:
             "documenten": self.cleaned_data["documenten"],
             "kownslReviewRequestId": self.cleaned_data["review_request"],
         }
+
+    def on_submission(self):
+        if self._review_type is not None:
+            review_request = create_review_request(
+                self.zaak_url,
+                review_type=self._review_type,
+                num_assigned_users=len(self.cleaned_data["users"]),
+            )
+            self.cleaned_data["review_request"] = str(review_request.id)
+        else:
+            raise NotImplementedError(
+                f"Variable _review_type needs to be defined in {self.__name__}"
+            )
```

Comment:
This defensive style goes a bit against the zen of python (better to ask forgiveness than permission), so I'd replace that with:

```suggestion
        assert self._review_type, "Subclasses must define a '_review_type'"
        review_request = create_review_request(
            self.zaak_url,
            review_type=self._review_type,
            num_assigned_users=len(self.cleaned_data["users"]),
        )
        self.cleaned_data["review_request"] = str(review_request.id)
```